ruleset/rules/0750-github_rules.xml

```diff
@@ -0,0 +1,3225 @@
+        <description>GitHub Project delete.</description>
+        <options>no_full_log</options>
+        <group>git_project</group>
+    </rule>
+
+    <!-- project.unlink -->
+    <rule id="91265" level="7">
+        <if_sid>91259</if_sid>
+        <action>project.unlink</action>
+        <description>GitHub Project unlink.</description>
+        <options>no_full_log</options>
+        <group>git_project</group>
+    </rule>
+
+    <!-- project.update_org_permission -->
+    <rule id="91266" level="5">
+        <if_sid>91259</if_sid>
+        <action>project.update_org_permission</action>
+        <description>GitHub Project update org permission.</description>
+        <options>no_full_log</options>
+        <group>git_project</group>
+    </rule>
+
+    <!-- project.update_team_permission -->
+    <rule id="91267" level="5">
+        <if_sid>91259</if_sid>
+        <action>project.update_team_permission</action>
+        <description>GitHub Project update team permission.</description>
+        <options>no_full_log</options>
+        <group>git_project</group>
+    </rule>
+
+    <!-- project.update_user_permission -->
+    <rule id="91268" level="5">
+        <if_sid>91259</if_sid>
+        <action>project.update_user_permission</action>
+        <description>GitHub Project update user permission.</description>
+        <options>no_full_log</options>
+        <group>git_project</group>
+    </rule>
+
+    <!-- organization_domain
+            create
+            delete
+    -->
+    <rule id="91269" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^organization_domain.\.*</action>
+        <description>GitHub Organization domain.</description>
+        <options>no_full_log</options>
+        <group>git_organization_domain</group>
+    </rule>
+
+    <!-- organization_domain.create -->
+    <rule id="91270" level="5">
+        <if_sid>91269</if_sid>
+        <action>organization_domain.create</action>
+        <description>GitHub Organization domain create.</description>
+        <options>no_full_log</options>
+        <group>git_organization_domain</group>
+    </rule>
+
+    <!-- organization_domain.delete -->
+    <rule id="91271" level="5">
+        <if_sid>91269</if_sid>
+        <action>organization_domain.delete</action>
+        <description>GitHub Organization domain delete.</description>
+        <options>no_full_log</options>
+        <group>git_organization_domain</group>
+    </rule>
+
+    <!-- private_repository_forking
+            enable
+            disable
+    -->
+    <rule id="91272" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^private_repository_forking.\.*</action>
+        <description>GitHub Private repository forking.</description>
+        <options>no_full_log</options>
+        <group>git_private_repository_forking</group>
+    </rule>
+
+    <!-- private_repository_forking.enable -->
+    <rule id="91273" level="5">
```

Comment:
Looks like enabling forking of a private repository should have higher level than disabling forking on a private repository

Reply:
Done